getting-started/quick-start.md

```diff
@@ -20,15 +20,25 @@ If you already have `crystal`, `amber`, and a database installed, you can [skip
 
 ### 1.1 Install `crystal` and `amber`
 
-Instructions for OS X using homebrew and Debian/Ubuntu are below. See full installation instructions [here](https://crystal-lang.org/docs/installation/) for RedHat & CentOS, ArchLinux & Derivatives, and more complete instructions.
+Instructions for macOS using Homebrew or MacPorts and Debian/Ubuntu are below. See full installation instructions [here](https://crystal-lang.org/docs/installation/) for RedHat & CentOS, ArchLinux & Derivatives, and more complete instructions.
 
-#### OS X with homebrew
+#### macOS
 
-Installing Amber with Homebrew also installs crystal.
+- Homebrew
 
-```text
-brew install amber
-```
+  ```text
+  brew install amber
+  ```
+
+  Note: This also installs the Crystal Programming Language.
```

Comment:
this note doesn't need to be repeated, but could be provided above, near the title of the section.

Reply:
Will update.